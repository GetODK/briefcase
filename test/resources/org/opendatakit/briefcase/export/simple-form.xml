<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml">
  <h:head>
    <h:title>Simple form</h:title>
    <model>
      <instance>
        <data id="simple-form">
          <meta>
            <instanceID/>
          </meta>
          <some_text/>
        </data>
      </instance>
      <itext>
        <translation lang="English">
        </translation>
      </itext>
      <bind nodeset="/data/meta/instanceID" type="string" readonly="true()" calculate="concat('uuid:', uuid())"/>
      <bind nodeset="/data/some_text" type="string"/>
    </model>
  </h:head>
  <h:body>
    <input ref="/data/some_text"/>
  </h:body>
</h:html>