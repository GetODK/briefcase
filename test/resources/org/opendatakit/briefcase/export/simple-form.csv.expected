SubmissionDate,meta-instanceID,some_text,KEY
"Apr 1, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000090
"Apr 2, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000091
"Apr 3, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000092
"Apr 4, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000093
"Apr 5, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000094
"Apr 6, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000095
"Apr 7, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000096
"Apr 8, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000097
"Apr 9, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000098
"Apr 10, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000099
"Apr 11, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000100
"Apr 12, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000101
"Apr 13, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000102
"Apr 14, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000103
"Apr 15, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000104
"Apr 16, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000105
"Apr 17, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000106
"Apr 18, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000107
"Apr 19, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000108
"Apr 20, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000109
"Apr 21, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000110
"Apr 22, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000111
"Apr 23, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000112
"Apr 24, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000113
"Apr 25, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000114
"Apr 26, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000115
"Apr 27, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000116
"Apr 28, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000117
"Apr 29, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000118
"Apr 30, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000119
"May 1, 2018 12:00:00 AM",,some value,uuid00000000-0000-0000-0000-000000000120
